### PR TITLE
New version: Friction.Friction version 0.9.6.1

### LINKED ISSUE #196208: [Update Request]: Friction.Friction 0.9.6.1
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Friction.Friction

### Package Version

0.9.6.1
~1.0.0-beta2~

### Please describe the changes you would like to see

https://friction.graphics/releases/friction-0961.html  
~https://friction.graphics/releases/friction-100-beta2.html~

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Related (**but failed**) #196208 

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?
  - [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

image
image

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/196483)

### CODE REVIEW

**Comment by stephengillie** [comment]
URL: https://github.com/friction2d/friction/releases/download/v0.9.6.1/friction-0.9.6.1-setup-win64.exe 
Status Code: 200

(Automated message - build 900)

File: manifests/f/Friction/Friction/0.9.6.1/Friction.Friction.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
...HA256 Checksum: \n0380cedd44cacdfeec9fed886ea39cbc8f5bf18c492756b0d93e7d603a065619  friction-0.9.6.1-setup-win64.exe\n\nSHA256 Checksum: \n2a2c092cab25982cf2dae0c2a928dbe2d4bc03cc569ef217abdd47b2df4b5dcb  friction-0.9.6.1-windows-x64.7z\n\nSHA256 Checksum: \n26c6f9f07c8f9295be67005095fa8962d105844ba03f2cc3b917c6760b9b3c4d  friction-0.9.6.1.tar.xz\n\n-----BEGIN PGP SIGNATURE-----\n\niQIzBAEBCgAdFiEEaJcvKbgc2jraF+repD5Lhhb4ydMFAmbGPJoACgkQpD5Lhhb4\nydOtuQ/9E/OGypDwVuVa0LbIYMf+hSz9CEG0Rucfi08YJMCyixcR46Ly7TZc7inH\nHHkNFZu4e3vKN8n52gIndPxMIDixRIUkCGzUPjiT1bghYYrI4drULs2NuahKUOng\nUX1DhQdg7zFoOuFWpCzKChUG/WTwQfiA4ZLRFMyKGbmpmAsTWDs8IfzSld83zM5Y\nCkoxxg0IBvL/69iQs8HfHcY2kv651KBtoG81Prv3OfcjzsFdIgXJqllwBfFdEguz\nVHTiAo6zM4BqykU/lACG3nzTiCIom+faXmMr6EUodjXX1K9mXdYlie/3aDt2TuFr\nBJrBSBZdOA6gKB7U51od4aReWnWfBEKQQhvS+V/E529KS9eaazAzpsLKqJE/lpfW\n0HDGtUuCeC/F57MMo/K129TezYUOZy8NL1LDtKw+etnHRxDVOM0bumT1bX96JlWy\nIB5Ioc9pTX0ndehBSy0zdF+w3aIrH9/qVndveXWYSG4RucBgPmM4LCxOwVQG4hqX\nBIolT9iqYLuI8+slxY7mqSe3pO9lNIwQI6WgvhiaPuAQupBRFwkvYDLKeiqqKbbk\nlIqOiJNcat0NhCyQCrQ1EiLIrRouI0DDw1zbzaHvasfpI99p0sGjWEMFP3NfTAoW\nkT+XT6hGx8DwaUNQVv9d6DXWQfvGeJJoWFbr8/nWOGXigwNUjNk=\n=vHHX\n-----END PGP SIGNATURE-----"
```
Comment (Dragon1573) [inline manifests/f/Friction/Friction/0.9.6.1/Friction.Friction.locale.en-US.yaml:24]:
```suggestion
ReleaseNotes: |-
  The first maintenance release for 0.9.6 is now available.

  This release marks the end of development for the 0.9 series. The 0.9 series is now in maintenance mode, receiving only important bug fixes until the upcoming 1.0 release later this year.

  See release notes for more information.
```

File: manifests/f/Friction/Friction/0.9.6.1/Friction.Friction.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ReleaseNotesUrl: https://github.com/friction2d/friction/releases/tag/v0.9.6.1
+Documentations:
+- DocumentLabel: Linux | Developers
+  DocumentUrl: https://friction.graphics/documentation/source-linux.html
+- DocumentLabel: Shader Effects (Draft) | Users
+  DocumentUrl: https://friction.graphics/documentation/shaders/
+- DocumentLabel: Shortcuts (Draft) | Users
+  DocumentUrl: https://friction.graphics/documentation/shortcuts.html
+- DocumentLabel: Windows | Developers
+  DocumentUrl: https://friction.graphics/documentation/source-windows.html
```
Comment (Dragon1573) [inline manifests/f/Friction/Friction/0.9.6.1/Friction.Friction.locale.en-US.yaml:34]:
```suggestion
```

Can't find any documentation URL from the site.

**Comment by Dragon1573** [comment]
> [!IMPORTANT]
>
> This package seems unable to be installed properly in _Microsoft Azure Pipelines_. Stop further attempts.

**Comment by luzpaz** [comment]
@Dragon1573 quick question, the URLs above are from within the Friction application ? If not, where is there source from ?

**Comment by Dragon1573** [comment]
> The URLs above are from within the Friction application ? If not, where is there source from ?

Don't know. :disappointed: Just inherit from previous versions using _Komac_.

**Comment by luzpaz** [comment]
@Dragon1573 those URL were removed since then. So that shouldn't be a blocker

**Pull request closed without merging**